\chapter{Introduction}
\graphicspath{ {C:/Users/giaccoyu/Desktop/TeXWorks/Fig/Intro/} }
This is the part of Introduction.
\section{ER stress and UPR}
\subsection{The overview of ER in the cell}
Endoplasmic Reticulum (ER) is one of the key organelle in eukaryotic cells. ER has a very big surface area, can reach up to 30 times bigger than the plasma membrane. That fact indicates that ER has a highly folded structure and closely associated with its function in the cell. ER is very important for production and processing of biomolecules like proteins and lipids and ER is found to connect to the nuclei.  Basically, there are two types of ER, rough ER (RER) and smooth ER (SER), depending on whether there are ribosomes on ER surface. With ribosomes on the membrane, the rough ER provides the circumstance for protein synthesis because ribosomes are the place for DNA translation and produce the corresponding peptides. The smooth ER is the place for lipid synthesis like the production of fatty acids, steroids and glycerolphospholipids. Additionally, the smooth ER has the exit site, because the transport vesicles containing the lipids or proteins will detach and move towards Golgi apparatus from these sites. For this reason, the smooth ER usually locates outside of the rough ER.\\
\begin{figure}
	\centering
	\includegraphics[width=7cm]{ER.jpg}
	\caption[Illustration of two types of endoplasmic reticulum.]
	{This figure shows the general illustration of endoplasmic reticulum. a. The rough ER with many ribosomes (red dots) stubbed on the surface. b. The smooth ER does not have ribosome on the surface. c. The nuclei of the cell is surrounded by ER.}
\end{figure}
For the protein synthesis in rough ER, the whole process from the beginning of synthesis to protein secreting is very complicated. All the proteins start from amino acids and different amino acids are carried by tRNA based on the matured RNA sequence to join as a peptide chain. This process is executed on ribosomes on the rough ER. Though the synthesis of a whole amino acid chain is only the first step. A fully functional protein is not only depends on its linear amino acid sequence, which is also named as its primary structure, but also depends on its spatial structure. The folding process include many complicated mechanisms like amino acid modification, which is also named as post-translational modification since this process happens after the synthesis of polypeptide chains. Modifying the nascent polypeptide does make the whole structure more flexible and let the whole polypeptide easier to form a highly folded structure. Researchers have identified many types of molecular modification during this step, like terminal modifications, sequence addition and deletion, functional group modification on individual amino acids and disulfide bridge formation. 
\subsection{Post-translational Modification}
In detailed synthesis mechanism, during the assembly of polypeptide, a short signal sequence is added on the terminal of the peptide chain. Generally the length of those signal peptides are quite short, usually from 10 to 30 amino acids. However, functionally those short signal peptides are very important in protein synthesis because it can be detected by some signal recognition particles (SRP) on the ER membrane and thus help the whole peptide reside on the ER membrane for further structural processing. \\
After the nascent polypeptide is anchored on ER membrane, the signal sequence will be cleaved from certain sites by some proteases. Then on the \textit{N}-terminal, the nascent peptide will be glycosylated. The glycosylation has a very conservative core oligosaccharide structure in most secreting proteins. This structure contains several branches and compromised with three types and totally fourteen saccharide risidues. The core structure is illustrated in Figure \ref{fig:glycosylation}. The three saccarides are glucose, mannose and \textit{N}-Acetylglucosamines, and thus the fourteen saccharide structure can also been written as \ce{Glc3Man9GlcNAc2}). The expression also indicates the number for each type of saccharide. Generally, the glycosylation starts from two \textit{N}-Acetylglucosamines being transferred to dlichol phosphate. Dolichol phosphate acts as a donor in the building process of this oligosaccharide core structure. From the beginning, the addition of GlcNAc requires UDP-GLcNAc. This step is quite important and we have further discussion in the following part. Then five UDP-Man (No. 3-7 in \ref{fig:glycosylation}) add on the second GlcNAc and start forming different branches. Until now, the glycosylation process occurs on the cytosolic side of ER. Then the incomplete will flip into ER lumen for further saccharide units addition with the assistance of specific transferase. Once the oligosaccharide structure is complete, it will be transferred to an asparagine residue on a protein. 
\begin{figure}[h]
	\centering
	\includegraphics[width=5.5cm]{glycosylation}
	\caption[Core 14 saccaride struture during protein glycosylation.]
	{This figure shows the core structure (\ce{Glc3Man9GlcNAc2}) of the glycosylation mode on the N-terminus of the polypeptide chain in most proteins. Glycosylation process starts from the bottom (\textit{N}-terminal of the nascent peptide) and begins with adding 2 \textit{N}-Acetylglucosamines (GlcNAc). Then 9 mannoses follow the \ce{GlcNAc} and splits into 3 branches A B and C. On the end of branch A, another 3 glucoses follow the mannose and make it the longest branch in this structure. \citep{howell2013endoplasmic}}
	\label{fig:glycosylation}
\end{figure}
\subsection{Protein Structure and Folding}
Protein folding is a necessary way to turn the linear nascent proteins to three-dimensional functional ones for almost all the proteins in cell. According to the definition of protein structure, there are at most four levels describing the general folding degree, from primary to quaternary structure, as showed in Figure \ref{fig:protein_structure_level}.\\
The primary structure represent the amino acid sequence in the polypeptide, then hydrogen bonds can be easily established between certain amino acid residues to from comparatively stable helix ($\alpha$-helix) or pleated sheet ($\beta$-sheet) structure and this is regarded as the secondary structure of protein. Tertiary structure refers a three-dimensional single peptide structure comprising those basic helices and pleated sheets after folding. The peptide with tertiary structure has biological activities to some extent. Moreover, for many proteins, there is a even higher level of folding, which contains multiple tertiary structure subunits and the whole protein is arranged by non-covalent bonding among those monomers. Those proteins, enzyme complex for example, are fully functional only in this case.
\begin{figure}[h]
	\centering
	\includegraphics[width=12cm]{protein_structure_level}
	\caption[The four levels of protein structure.]
	{The illustration defining the four levels of protein structure. The primary structure refers the polypeptide chain after the translation on the ribosome. Then the sequence of polypeptide and some modifications on the amino acid residues help the peptide chain form rigid local structures and they are defined as the secondary structure of protein. Among those secondary structures, $\alpha$-helix and $\beta$-pleated sheets are the most common ones. Then the polypeptide chains go further folding to form a more complicated structure, and the overall three-dimensional arrangement of all the whole peptide is defined as tertiary structure. For protein complex, they are assemblies of multiple polypeptides via non-covalent bonds. Each polypeptide acts as a subunit and it collaborate with other ones to make the whole protein fully functional \citep{silva2014protein}.}
	\label{fig:protein_structure_level}
\end{figure}
The peptide folding itself from primary structure to tertiary structure contains a series of very comprehensive physical process. As our previous presentation, many different types of modification on the amino acid residues are prerequisites to form a highly folded structure. Those modifications create the opportunities for different amino acid residues to form hydrogen bonds or interact via Van der Waals forces. One opinion is that there are some 'key' residues on the polypeptide chain and those residues could be determinant factor of forming the native topological structure after millions of stochastic interaction between different residues on the peptide chain, with reviewing the statistical simulation of a large scale of proteins with different length \citep{dobson2003protein}. Moreover, although not clear, people suggest that the proteins are folded in small modules rather than from begin to end.\\ 
However, not all the proteins are folded spontaneously, some types of proteins known as molecular chaperones can facilitate helping the proteins be folded in a more efficient way \textit{in vivo}. There are some well-studied chaperone families like Heat Shock Proteins (Hsp). In more details, there are a few different Hsp families like Hsp40, Hsp60, Hsp70 and Hsp90. The number of those heat shock proteins indicates their approximate molecular weight corresponding to the different families. Although the exact function of those families varies in locations and steps in protein synthesis and folding, they help prevent the aggregation of unfolded proteins in the cell. The molecular chaperones can mediate the energy landscapes of the proteins and make them easier to be folded. A certain type of chaperones belong to the Hsp70 family called 78kD Glucose-Regulated Protein (Grp78) is located in the lumen of ER \citep{munro1986hsp70}. Grp78 is also called Binding Immunoglobulin Protein (BiP). The BiPs can also assist the translocation process of secretory proteins, since \citet{vogel1990loss} found that the secreting was blocked without the existence of BiP. Functionally, BiP and other Hsp70 has binding site for ATP and thus they have the biological activity of ATPase \citep{chappell1986uncoating}, and that offers chances for BiP to lower the energy requirement for protein folding. In fact, BiP can recognise the hydrophobic regions in the unfolded peptides. Those hydrophobic regions are usually hidden inside of the unfolded peptides and people found that the binding and release by BiP are ATP-dependent process \citep{bukau1998hsp70}. 
\subsection{What is ER Stress?}
ER stress describes the situation where unfolded or misfolded proteins accumulate in the ER lumen. In normal condition, cell has a few different ways to control the protein quality during the synthesis. For the misfolded proteins in ER, there is a basic quality control mechanism to prevent their accumulation. Those proteins can be distinguished after a series of glycosylation and deglycosylation reactions. Misfolded proteins, once identified by the mechanism, will be exported to cytosol via some protein channels. They will finally be ubiquitinated and then degraded by proteasomes. Tagging ubiquitin on the mis-folded proteins requires a cascade of enzymes including Ub-activating enzyme (E1), Ub-conjugating enzyme (E2) and Ub-protein ligase (E3). Usually those enzymes are in a complex. This process is also called ER Associated protein Degradation (ERAD). The recycling process of ERAD can be very fast, from a few minutes to hours \citep{schubert2000rapid}.
\begin{figure}[h]
	\centering
	\includegraphics[width=8cm]{misfold_control}
	\caption[The general protein quality control process in endoplasmic reticulum.]
	{The quality control mechanism flow in ER lumen. The nascent polypeptide will be folded there. The correctly-folded proteins will finally be transported to Golgi via small vesicles. Those vesicles derive from the ER membrane on its exit sites. However, mistakes occur during the protein folding process and misfolded proteins appear in ER lumen. Those misfolded proteins will be processed in another pathway and the ER tries to clear those mistakenly folded proteins. They will be exported from certain channels on ER membrane, and finally be tagged with ubiquitin to go degradation in the cytosol.}
	\label{fig:misfold_control}
\end{figure}
As we have already introduced the molecular chaperones in the previous section, in the ER lumen, Grp78, or BiP can act as modulators in controlling the protein folding quality. Because BiP can facilitate the folding process of nascent proteins in ER, so finally BiP keeps the total amount of unfolded proteins under a certain level. In normal cell growth condition, BiP is abundant for protein folding in ER lumen. However, its expression will be significantly induced when unfolded proteins accumulate. This can be regarded as a self-defensive mechanism since accumulation of unfolded proteins represents the increase of ER stress, and the role of BiP is buffering that stress. \\
BiP can also try to help the misfolded proteins to be refolded, however the refolding process is blocked due the unusual conformation of those misfolded proteins. BiP will keep binding on the misfolded ones and retain in the lumen of ER, until being degraded \citep{knittler1995molecular}. In ERAD, BiP is also found as an important regulator. \citet{nishikawa2005roles} have found BiP participated in the mis-folded protein degradation in ERAD-C pathway. The ERAD-C describes the ERAD for the mis-folding in cytosolic domains of of the transmembrane proteins. Other two types of ERAD are ERAD-L and ERAD-M, indicating the degradation of the proteins with mis-folded lumenal domains and transmembrane domains (TMDs) respectively. For ERAD-L, people also found the possible ERAD regulatory mechanism of BiP from yeast model, since the interaction between ERAD-L substrate CPY* (Mutated form of carboxypeptidase Y) can be recognised by Kar2 (The homologue of BiP in yeast cell) \textit{in vivo}.\\
In mammalian cell research, ER stress is believed to link with a series of diseases because of the dysfunction over-accumulation of mis-folded or unfolded proteins in ER lumen. %Describing the ER stress-related disease%
\subsection{Inhibition of \textit{N}-Glycosylation}
In the synthesis of functional protein, 
\subsection{Unfolded Protein Response}
The Unfolded Protein Response (UPR) is designated as the responsive mechanism to ER stress in the cell, especially to the over-accumulation of unfolded protein in ER lumen. UPR itself is a systemic signalling in the cell to prevent it go dysfunctional for lacking the fully functional proteins.\\
Previously we discussed briefly about the biological function of a certain family of molecular chaperones Hsp70 in ER lumen, and BiP is a typical class of Hsp70 which can detect and help the unfolded proteins to be refolded for its ATP-transferase feature. There are also evidences showing the regulatory effect in ERAD under stress because it can keep binding on those mis-folded proteins until they are ubiquitinated. The fact is that BiP belongs to a group of genes induced in the UPR signalling. \\
The ER stress and UPR topic was firstly discussed in mammalian cell. Currently people have already identified three different UPR pathways depending on what type the ER stress sensor is. Although those sensors behaves differently in transducing the stress signals, they are all located on the ER membrane.\\
The first type of ER stress sensor is called Inositol-Requiring protein, or Inositol-Requiring Enzyme 1 (IRE1). IRE1 has two isoforms in the cell, IRE1$\alpha$ and IRE1$\beta$. Based on the results tested in different mouse tissue, \citet{bertolotti2001increased} found that IRE1$\alpha$ expressed ubiquitously, while IRE1$\beta$ only expressed in the cells from certain tissues like stomach and colon cells. So, for general research for IRE1 pathway in UPR siganlling, IRE1$\alpha$ should be a more representative protein than its homologue. Functionally IRE1 is a kinase and also a endoribonuclease. It can form a dimer after auto-phosphorylation and it has mRNase domian on its cytosolic side, as IRE1 is a transmembrane protein \citep{prischi2014phosphoregulation}. The phosphorylated dimer is the activated form of IRE1 when it senses ER stress in the lumen side and ready to catalyse certain mRNA in the cytosol. The substrate of activated IRE1 is identified as the mRNA of X box Binding Protein 1 (XBP1), and totally 26 nucleotides-long intron region will be removed inside the mRNA sequence after catalysation. The whole catalysing process results the reading frame shift and it is essentially RNA splicing. In the UPR signalling, the splicing of XBP1 mRNA is a very important step because the spliced XBP1 mRNA can then initiate the translation of XBP1, which is an important transcription factor in UPR. XBP1 transcription factor will then translocate into nucleus to upregulate the expression of a series of UPR genes. The ER-located molecular chaperone gene bip is one of the UPR genes initiated by XBP1, but results shows that the expression of BiP is only modestly controlled by XBP1 \citep{lee2003xbp}, indicating BiP is the product of other signalling pathways. Moreover, on the lumen side of IRE1, people have identified the BiP binding sites and found that BiP attaches on the luminal domains under unstressed condition \citep{kimata2004role}. The binding of BiP sequester the inactive IRE1 and then inhibit oligomerisation of IRE1 or facilitate de-oligomerisation and deactivation of the activated IRE1 molecules \citep{pincus2010bip}. After sensing ER stress, BiP dissociate from the binding sites to bind to mis-folded or unfolded protein in the lumen of ER and lead the conformational change of the IRE1 and activate the downstream signalling in UPR. Thus BiP acts like a controlling element switching IRE1 on or off. \\
Another type of ER stress sensor is activating transcription factor 6 (ATF6). The name of ATF6 indicating its function as a transcription factor, with a basic leucine zipper (bZIP) structure. The mechanism of ATF6 induced UPR signalling is quite different with the IRE1 pathway because previous works showed ATF6 would be translocated into nucleus with ER stress induction. Furthermore, ATF6 on the ER has different molecular weight with the ones in the nucleus. The molecular weight of the inactivated ATF6, which is located on ER membrane, is around 90kD, and ATF6 in nucleus is only around 50kD \citep{haze1999mammalian}. The size changing refers a protein cleavage process during the UPR. The proteolysis finally cut on the cytosolic side of ATF6 and preserve the bZIP structure at the same time. \citet{ye2000er} found the enzymes conducting the further proteolysis are Site-1 Protease (S1P) and Site-2 Protease (S2P). S1P and S2P are previously found in charge of processing the cleavage of Sterol Regulatory Element Binding Proteins (SREBPs). The cleavage of SREBPs by S1P and S2P is in Golgi lumen. Also, \citet{shen2002er} found two Golgi Localisation Signals (GLS) in ATF6 sequence and ATF6 keeps those GLS after being cut from ER lumen. Another fact is, before dissociation from ER membrane, BiP is also binding to ATF6 on its ER lumen side , which is similar with IRE1, but in a relatively more stable way, which means BiP cannot switch ATF6 on or off \citep{shen2005stable}. Because of the mobility of ATF6, once it is activated under ER stress, it leaves ER and enters nucleus via Golgi, the total process is not revertible. After cleavage by S1P and S2P in Golgi apparatus, ATF6 becomes functional transcription factor and go into nucleus. In the set of ATF6 target genes, BiP is one important gene which is significantly induced under ER stress \citep{adachi2008atf6}. \\
The third receptor is Protein kinase RNA-like ER Kinase (PERK) \citep{shi1998identification}. Like IRE1, the activation of PERK requires dimerisation of the amino-terminal ER luminal domains (NLDs) of the molecules on the ER membrane \citep{liu2000ligand}. Activated PERK can further regulate the phosphorylation of eukaryotic initiation
factor-2$\alpha$ (eIF2$\alpha$) because the kinase region is located on the cytosolic side of PERK \citep{harding1999erratum}. The kinase has its activity when PERK dimer is formed and then transfer phosphates on eIF2$\alpha$. In the cell, the role of eIF2$\alpha$ is protective and \citet{boyce2005selective} found that using specific inhibitor to prohibit the dephosphorylation of eIF2$\alpha$ can save the cell from ER stress, that indicates only phosphorylated eIF2$\alpha$ can contribute to be cytoprotective element. Also the phosphorylated eIF2$\alpha$ can generally reduce the initiation of mRNA translation, but on the opposite, at the same time, facilitate the translation process for a few specific mRNAs. Those mRNAs contain upstream open reading frames (uORFs), under normal condition, while there are abundant unphosphorylated eIF2$\alpha$, the complete translation of that mRNA will be blocked \citep{baird2012eukaryotic}. One mRNA encoding activating transcription factor 4 (ATF4) is one of those special mRNAs. ATF4 is similar with ATF6, also has a bZIP transcription factor structure, and can initiate the expression of a series of stress response genes. C/EBP-homologous protein (CHOP) and growth arrest DNA-inducible gene 34 (GADD34) are two important target of ATF4 and can further influence the PERK response pathway by positively regulating some phosphatase complexes like protein phosphatase 1C (PP1C) \citep{harding2003integrated,novoa2001feedback}. Those phosphatase complexes will in turn facilitate dephosphorylation of those phosphrylated eIF2$\alpha$ to form a negative loop in the signalling. Other than CHOP and GADD34, other stress-induce genes like DNA damage inducible transcript 4 (DDIT4, or REDD1) are cytoprotective elements \citep{whitney2009atf4}. Also, since there is bZIP structure on ATF4 protein, it can form either homo-dimer or hetero-dimer when transcribing target genes, indicating there is possible overlapping targets shared with other transcription factors, especially ones with bZIP structure as well \citep{hai1989transcription}.\\
The UPR mechanism is in brief conservative across the species in different kingdoms. In invertebrate cells, the UPR signalling is highly similar with the one in mammalian cells, because the homologues of all the three ER stress sensors can be identified. However, in fungal cell, the UPR is much simplified. Currently, only the homologue of IRE, which is called ire1p, is found in \textit{Saccharomyces cerevisiae}, indicating that IRE1 pathway could be the most basic and conservative UPR mechanism in eukaryotic cells \citep{patil2001intracellular}. In yeast cells, the signalling induced by activated ire1p is similar, involving the splicing process of mRNA. The substrate for splicing in yeast cells is called HAC1, the spliced HAC1 can also become a transcription factor and initiate the transcription of a group of UPR genes, including BiP \citep{foti1999conservation}. For that reason, HAC1 is thought to be the homologue of XBP1 in mammalian cells, and the share the bZIP structure at the same time. \\
The UPR signalling in plant cells has the complexity between animal cells and fungal cells in general, because previous researches have confirmed the homologues of IRE1 and ATF6 exist in plant cells but the homologue of PERK is currently missing in the whole signalling pathways \citep{howell2013endoplasmic}. As the result, currently we think that there are two types of ER sensor on the membrane, the IRE1-like element and ATF6-like element.\\
Similar to mammalian cells, IRE1 in plant cells also has two different paralogues AtIRE1$\alpha$ (At2g17520) and AtIRE1$\beta$ (At5g24360) \citep{moreno2012ire1}. For convenience, we use IRE1 to represent the protein in plant cells only in the following text. Under ER stress, dimerisation of IRE1 is also required in plant cells and then the autophosphorylation brings the IRE1 dimer catalysing activity \citep{lee2008structure}. The activated IRE1 on the ER membrane can then conduct splicing of the nascent mRNA, which also encode bZIP and a transcription factor region. The mRNA is identified as basic leucine zipper 60 (bZIP60). By analysing the sequence of XBP1, HAC1 and bZIP60, \citet{zhang2016divergence} showed the conservation at those key regions, the bZIP encoding region and N-\ce{x7}-R/K DNA binding motif. During the splicing, a 23-nucleotides long intron in the middle of bZIP60 mRNA will be cut by phosphorylated IRE1. The length of removed intron is quite closed to the ones on XBP1 in mammalian cells, which is 26 nucleotides long. With the insight of the bZIP mRNA structure, the 23 nt intron is located between two hairpins, and they will merge into a new hairpin loop structure \citep{nagashima2011arabidopsis}. Like XBP1, spliced bZIP60 will become a transcription factor and initiate transcription of UPR genes by binding the DNA chain with bZIP structure. The target UPR genes induced by bZIP60 includes BiP and protein disulfide isomerase (PDI) and calnexin (CNX). In more detailed research about the target that are modulated by bZIP60 transcription factor, \citet{iwata2008arabidopsis} found there was almost no BiP3 expression in \textit{bzip60} mutant whereas BiP1 and BiP2 showed little difference with wild type \textit{Arabidopsis} in gene expression. In \textit{Arabidopsis} genome database, by now three BiP genes paralogues are identified as BiP1 (AT5G28540), BiP2 (AT5G42020) and BiP3 (AT1G09080). Among these three BiP genes, BiP1 and BiP2 are highly similar in DNA sequence and the two genes share 97\% identity overall, at the same time, the identity between BiP1 and BiP3 is much lower, only 73\%, although the role in regulating the unfolded proteins in ER of the three BiPs are similar. However, currently there is no clear evidence showing any type of those BiPs binds to IRE1 on its luminal side without detecting unfolded protein, but the analysis of the sensor domain on IRE1 indicates the mechanism could be similar as it in mammalian cells \citep{koizumi2001molecular}.\\
\begin{figure}
	\centering
	\includegraphics[width=12cm]{AtIRE1}
	\caption[The mode of IRE1 signalling pathway in Plant cell UPR.]
	{The IRE1 pathway in plant cell UPR signalling. IRE1 is located on the ER membrane with stress sensor domain on its luminal side and ATP transferase/nuclease on its cytosolic side. Under ER stress, when the misfolded or unfolded proteins accumulate in ER lumen, IRE1 can form dimers by autophosphorylation and then change its conformation on the cytosolic side. That makes the IRE1 dimer a nuclease and then catalysing the splicing of \textit{bzip60} mRNA. The splicing removes a 23 nt intron between the two hairpins on \textit{bzip60} mRNA. Spliced mRNA can be translated into bZIP60, which is a transcription factor. After becoming a transcription factor, bZIP60 can induce the expression of a series of UPR genes to protect the cell from ER stress.}
	\label{fig:AtIRE1_intro}
\end{figure}
The ATF6-like ER stress sensor in plant cells has a few members, but all of them encodes bZIP transcription factor as well. Researchers have already found at least three bZIP paralogues: bZIP28 (AT3G10800), bZIP49 (AT3G56660) and bZIP17 (AT2G40950). bZIP28 is currently the most well-studied ATF6-like sensor in plant cells. The higher expression than other two in plant tissues make bZIP28 possibly be the predominant ATF6-like sensor in plant cell UPR. \citet{liu2007endoplasmic} also found similar cleavage of bZIP28 by S1P and S2P to get activated as a transcription factor, but only S1P cleavage site is now clear. An four-peptide long Arg-Arg-Ile-Leu (RRIL) cleavage site on the bZIP28 ER luminal side can be specifically recognised by S1P in Golgi \citep{srivastava2012elements}. \citet{liu2007endoplasmic} also found both BiP1/BiP2 and BiP3 were suppressed in two different \textit{bzip28} mutant, suggesting bZIP28 was more ubiquitous in controlling the BiP expression and BiP3 seemed to be regulated by both bZIP60 and bZIP28. The other two bZIPs, the knowledge is still limited, only \citet{liu2008salt} showed the cleavage of bZIP17 was induced when the plant was under salinity stress, and only S1P was confirmed in catalysing bZIP17 under stress \citep{liu2007salt}. The results may indicate bZIP28, bZIP49 and bZIP17 response to different types of ER stress. \citet{liu2010bzip28} tested the homodimerisation and heterodimerisation among bZIP60, bZIP28, bZIP49 and bZIP17 in their research. The two hybridisation in yeast showed bZIP28 tended to homodimerise but one could dimerise with any among bZIP28, bZIP49 and bZIP17. However, bZIP60 tended to form heterodimers with bZIP28 and bZIP17. That could possibly explain why bZIP60 and bZIP28 had similar controlling effect for BiP3, but only bZIP28 could significantly regulate the transcription of BiP1 and BiP2 rather than bZIP60. \citet{liu2010bzip28} also found the target UPR genes of bZIP28 requires both CCACG and CCAAT binding boxes. CCACG box is the binding target of bZIP dimer, either bZIP28 homodimer or bZIP heterodimer. CCAAT is the binding site for Nuclear transcription factor Y (NF-Y) heterotrimeric complex. That complex has three different groups of subunits: NF-YA, NF-YB and NF-YC, and totally there are 36 genes in \textit{Arabidopsis} genome. However, in yeast three hybridisation system, the complex combined with NF-YA4 (AT2G34720), NF-YB3 (AT4G14540) and NF-YC2 (AT1G56170) is found to strongly interact with bZIP28. Further quantitative PCR results showed only NF-YC2 is strongly induced under ER stress, suggesting NF-YC2 could also be a possible UPR gene regulated by bZIP28 \citep{liu2010bzip28}. Another feature shared with ATF6 is the on/off swtich controlled by BiP on the luminal side. The BiP binding domian on bZIP28 allows BiP binds to bZIP28 when there is no enhanced ER stress and dissociate from bZIP28 to bind to misfolded or unfolded protein under increased ER stress. Interestingly, BiP can also regulate the mobilisation of bZIP28 on ER membrane, even without ER stress induction, bZIP28 is found to mobilise from ER to Golgi and then to nucleus in \textit{bip} knockout mutant, but also the mobilisation is suppressed in BiP over-expressed mutant \citep{srivastava2013binding}. That gives us clear evidence that BiP is the key for stabilising bZIP28 on membrane and act as a switch in activating the downstream signalling in bZIP28 pathway.
\begin{figure}
	\centering
	\includegraphics[width=8cm]{bZIP28}
	\caption[Mode of bZIP28 signalling pathway in plant cell UPR.]
	{The bZIP28 pathway in plant cell UPR signalling. The purple arrows briefly illustrate the whole signal transducing process. There is a BiP binding site on the tail of bZIP28 C-terminal which is on the ER luminal side. Without ER stress, the BiP amount is abundant for refolding proteins or assistant in ERAD, at the same time, bZIP28 holds BiP and is stabilised on ER membrane. Under ER stress, while misfolded or unfolded proteins are overwhelmingly accumulated in ER lumen, BiP has priority to dissociate from bZIP28 to help those proteins get refolded. Once BiP dissociates from bZIP28, bZIP28 will leave ER membrane and mobilise with the assistance by some chaperones like Sar1 and re-locate on Golgi, possibly with some certain vesicle element proteins. On Golgi membrane, bZIP28 will be cleaved by S1P and S2P, though currently the precise S2P cleavage on bZIP28 is not clear. Cleaved bZIP28 will become transcription factor and initiate the expression a group of UPR gens include BiP, PDI, calnexin and possibly NF-YC2. NF-YC2 is one of the three subunits of NF-Y complex, which is required to cooperate transcribing UPR genes. NF-YA4 and NF-YB3 are the other two subunits, however, currently people only found NF-YC2 was upregulated by bZIP28.}
	\label{fig:bZIP28_intro}
\end{figure}
\section{The link between ER stress and cell death}
We have already had overview about how most proteins are synthesised in ER and what could happen if protein folding is interfered or even blocked. Like immune systems in human body, there is a complicated systemic defensive mechanism to protect the cells from ER stress. However, if ER stress is persisting or the amount of misfolded or unfolded protein keeps staying at very high levels, it will undoubtedly harm the growth of the cells, and even eventually be lethal for the cells, which is irreversible for the fate of cells. So, with the research on ER stress and more and more findings have already indicated the close relationships between ER stress, UPR and the death of cells.
\subsection{The Overview of Cell Death}
Firstly, we are going to introduce the basic concepts about cell death. Cell death describe the statue after ceasing biological functions and the cell can no longer maintain itself alive. Cell death can be caused by external physical damage or chemical substance infection, but in most cases, cell death contains sequences of programmed molecular events, that is collectively called Programmed Cell Death, or PCD in abbreviation. \\
Based on morphological or biochemical criteria, in current opinions, there are four different types of PCD in animal cells: apoptosis, autophagy, cornification and programmed necrosis, or necroptosis \citep{kroemer2009classification}.\\
Apoptosis is the major topic in animal PCD, and people started unveiling the secrets behind it since decades ago. In morphological analysis, \citet{kerr1972apoptosis} showed the different specimens describing how the cell separated, condensed, went fragmentation and then finally got digested in tumour tissue. Also, people realised that such kind of cell death was very common in development and believed that apoptosis is the way to control those over-produced cells, during the formation of hands and feet, for example. People then had gradually had more and more details about molecular signalling in different model organisms researches, but the core mechanism seemed highly conserved. A family of cysteine protease triggers a series of sequential molecular changes by cleaving at specific sites of a large number of intracellular proteins. The target cleavage site is usually an aspartic acid residue, so that cysteine protease family is also called caspase, where 'c' represents the cysteine family and 'asp-ase' indicates the specific cleavage site. There are seven different caspases in mammalian cells, but are divided into two subgroups depending on their roles in apoptosis process. One groups is called initiator caspase, including caspase-2, caspase-8, caspase-9 and caspase-10, they are normally inactive but can assemble into large caspase complex when triggered by apoptotic signals, becoming activated and then further activate executioner caspases. The executioner caspases is the another group of caspase, including caspase-3, caspase-6 and caspase-7. They are usually in dimers and cleaved by active initiator caspase to get activated themselves \citep{bao2007apoptosome}. Finally those activated executioner caspases can initiate cleavage of large variety of important proteins and bring the cell dramatic changes afterwards. \\
Autophagy is another important type in PCD, under microscope, autophagic dead cells can be clearly differentiated with apoptotic cells. Vacuolisation of cytoplasm is common and accumulated autophagic vacuoles can be found inside the cells. Vacuoles are the places where degradation and recycling of biomolecules or even organelles are processed. In current definition, there are three different types of autophagy based on how the degradation and recycling happen: macroautophagy, microautophagy and chaperone-mediated autophagy (CMA) \citep{parzych2014overview}. Macroautophagy is the major pathway in autophagy, the most typical thing during macroautophgy is the formation of autophagosome. Autophagosome is a double-layer membraned spherical structure. Autophagosome acts like a intracellular transportable vesicle and carries unwanted proteins and organelles to find and merge with lysosomes. The vacuoles spotted under microscope are the consequences after degradation by lysosomes. Also, the formation of autophagosome involves complicated molecular signalling. A family of autophagy genes (ATGs) can anchor on the phagophore membrane and play important roles in expanding the membrane and finally forming a close-up sphere. Microautophagy seems simpler because it does not have the participation of autophagosomes and the lysosomes can directly uptake the unwanted cargo from cytoplasm. But overall, people know very little about the possible signalling in microautophagy due to lacking reliable tools to investigate the process in details. Finally, the CMA process is now only been discussed in mammalian cells. As the name describes, CMA is associated with the activity of molecular chaperones. It is thought to be highly specific because people had already found Lys-Phe-Glu-Arg-Gln (KFERQ) motif on the proteins is required for recognition mainly by heat shock 70 kDa protein 8 (HSPA8/HSC70) \citep{dice1990peptide}. The recognised cytosolic proteins will then be transported into lysosomes for degradation. To sum up, although researches have classified different types of autophagy, however, the purpose of autophagy is still quite controversial. Autophagy is linked to both cell survival and cell death. On one hand, the degradation of organelles and proteins are prerequisites for cell death, but on the other hand, the degradation and recycling mechanism can provide extra nutrition resources to help the cell survive under harsh circumstances. So, generally, it is hard to use dichotomy to evaluate the role of cell autophagy \citep{bergmann2007autophagy, hippert2006autophagy}.\\
Programmed necrosis, or necroptosis alternatively \citep{kroemer2009classification}, is different with necrosis, since necrosis only describe the cell behaviours under accidental fatal extracellular stimuli. Although the morphological phenomenon is similar, like cell size enlarging, swelling of cytoplasmic area and swelling of some organelles. Also, the rupture of plasma membrane can easily be detected under microscope. But recent research works gave evidences that some necrosis can also be programmed by the cell. Receptor-interacting protein 1 (RIP1) is a important element in regulating programme necroptosis. RIP1 has serine/theorine kinase function, and it is found to interact with tumour necrosis factor  receptor (TNF) on cytoplasmic membrane. RIP1 can be activated by dimerisation and can interact with its homologue RIP3 to induce necrosis. Since RIP1 is also found to be implicated with apoptosis since RIP1 can form a complex by interacting with caspase-8 and trigger the caspase signalling cascade \citep{feoktistova2011ciaps}, so RIP3 it thought to conduct a switch-like controlling of programmed cell death between apoptosis and necroptosis \citep{zhang2009rip3}. Furthermore, some small molecules like necrostatin-1 (Nec-1) is found to inhibit the formation of RIP1/RIP3 complex and consequently inhibit cell necrosis \citep{degterev2013activity}, that result strengthen the idea that at least some necrosis phenomena are programmed by molecular events in the cell.\\
In fact, in the up-to-date definition, there is another highly specific type of cell death called cornification \citep{kroemer2009classification}. Cornification is location restricted because that type of cell death is specifically applied in epidermic cells, especially on terrestrial animal skins. The cornified cells are usually a dead cell layer covering the body, and they act like a protective barrier on the surface. The distinct feature of cornified dead cells is that lipids accumulate in both intra and extracellular space, which make the dead cell layer much harder than living cells. The cornification process is thought to begin with the degradation of molecules and organelles. After degradation, cathepsins released from lysosomes and \ce{Ca^{2+}} released from mitochondria and ER can activate transglutaminases. Transglutaminases are essential for forming cornified structure and linking it to lipid molecules crossing cell membrane. The extrusion of numerous lipids finally blocks water loss and forming a adamant cornified layer \citep{candi2005cornified}.\\
Overall, the three main programmed cell death classes can be distinguished using both morphological observation and testing the behaviours of important biochemical markers involved in the process. Additionally, we know there are some highly specific categories of PCD which can be only applied for certain types of cells, as we gave the example of cell cornification. However, the current PCD classification system is still not perfect because of the limitation from morphological study and being lacking of further knowledge about the molecular signalling. Also, people found the cross-talk phenomena between different types of PCD process, as we briefly discussed about the role of RIP1 in both apoptosis and necroptosis pathways for example. So more details should be introduced to improve the classification system, which can also give us more hints about the PCD mechanisms in plant cells.
\subsection{Programmed Cell Death in Plant Cells}
The death of plant cells shares some similarity with animal cells. The plant can also remove unwanted cells via PCD during development, growth and pathogenic resistance. One typical example is the leaf shape formation during development. The epidermal and mesophyll cells can be removed after programmed death and finally it gives the leaves specific shapes \citep{gunawardena2004programmed}. Another good example to show how pathogenic response can be related with cell death is, using tobacco mosaic virus to treat tobacco plants and finally the treated leaves has scattered yellow spots, indicating the dead cell areas, on their surface. But there are still many aspects which are different because the cell structure differs a lot. So for example, necrosis or necroptosis cannot be found in plant cells because the cell wall restrict the volume increase of cytoplasm. Cell wall is also rigid and thus it is unlikely to spot burst cells under microscope. So, it is necessary to build a specific classification system for plant cells. \\
In current opinions, one of the major difference between animal cell PCD and plant cell PCD is about apoptosis, because apoptosis is thought to be absent in plant cells. In animal model, cells go apoptosis will shrink and then break into multiple small parts as apoptotic bodies by self-separation. However, since the rigidity of plant cell wall structure, the cells cannot do the same as it in animal cells morphologically. Even without cell wall, the shrinkage of protoplasts under stress stimuli is not as same as it in animal cells during apoptosis, because integrity of plasma membrane cannot be kept in protoplasts. Moreover, in the view of biochemical behaviours, the link between caspase-like activity and apoptotic morphological behaviour is controversial  \citep{van2011morphological}. \\
So, how does the more precise classification system looks like in plant cell death? By now, there are two major classes: vacuolar cell death and necrosis, the definition is based on morphological observations and biochemical features \citep{van2011morphological}.\\
Vacuolar cell death describes the cell death related with vacuoles as the name. Vacuole is one of the typical organelles that is absent in animal cells. Usually it is the largest organelle in plant cells because it can take 30\% to 90\% of the total cell volume. Vacuole is the main storage for hydrolytic enzymes for recycling and it is also an important controller of osmotic pressure in the cell.  
\section{UPR and PCD, Cell Decision Making?}
%\section{SA signalling and its response}
\section{Related computational modelling works}
\section{Research objects}